pyartifactory/models/repository.py

```diff
@@ -114,6 +115,11 @@ class ContentSynchronisation(BaseModel):
     source: Source = Source()
 
 
+class FederatedMembers(BaseModel):
+    url: str = ""
+    enabled: bool = True

```

Comment:
Are you sure about that boolean type here ? In the official API , enabled is a string not a boolean (See [documentation](https://jfrog.com/help/r/jfrog-rest-apis/repository-configuration-json))